src/new-components/heroes/support.tsx

```diff
@@ -0,0 +1,58 @@
+import styled, { css } from 'styled-components';
+import { up } from '../../components/style-utils/breakpoint';
+import React from 'react';
+import { rgba } from 'polished';
+
+/**
+ * Provide the default space we want to give to heroes.
+ * All heroes should use this container.
+ *
+ * We use the beautiful technique of using the grid
+ * to spread the content over the full available width.
+ * That way we don't need any position absolute & friends.
+ * */
+export const HeroContainer = styled.div`
+  display: grid;
+  height: 520px;
+  ${up('md')} {
+    height: 640px;
+  }
+
+  overflow: hidden;
+  /**
+   make sure everything in the hero is always on its own stacking context
+   not to conflict with the global context
+   */
+  isolation: isolate;
+  position: relative;
+`;
+
+interface TextContainerProps {
+  dimmed?: boolean;
+}
+
+/**
+ * Combined with `HeroContainer` this will give any text the appropriate space
+ * within the hero aligned to the main content (by using the same grid template)
+ * The background can be optionally dimmed/
+ */
+export const TextContainer = styled.div<TextContainerProps>`
+  grid-area: 1/1;
+  display: grid;
+  grid-template-columns:
+    [main-start] minmax(24px, 1fr)
+    [content-start] minmax(0, 820px) [content-end]
+    minmax(24px, 1fr) [main-end];
+
+  > * {
+    grid-column: content;
+  }
+
+  align-items: end;
+
+  ${({ dimmed }) =>
+    dimmed &&
+    css`
+      background-color: ${rgba('#000000', 0.2)};
+    `}
```

Comment:
Based on a first experiment in figma, the text background can be dimmed for the text being readable:

![Screen Shot 2021-12-14 at 13 57 05](https://user-images.githubusercontent.com/1701755/146002554-9ab6016e-44d1-4088-bbc5-d7cb75dc77c1.png)